### DISCUSSION TITLE
Websocket support?

### DISCUSSION THREAD

**Discussion by indigo-jay** [cherrypy/cherrypy#1964] [Q&A] [answered]
Given that the [ws4py](https://github.com/Lawouach/WebSocket-for-Python) project has been archived, what if any plans are there to directly incorporate websockets into cherrypy? I've got an existing project using a very old version of cherrypy that I need to add websockets and a few other features, but I'm a little hesitant to spend the time doing the cherrypy update only to find that websockets are just not in the cards (or are still reliant on a seemingly dead project).

**Comment by webknjaz** [comment, marked as answer]
It was then unarchived later.
